source/agora/consensus/data/Block.d

```diff
@@ -817,3 +835,54 @@ version (unittest)
         return block;
     }
 }
+
+/// demonstrate signing two blocks at height 1 to reveal private node key
```

Comment:
I have a question. What if more than 1 node signs two different blocks at the same height? For example 10 nodes sign block A and 2 nodes (call them X and Y) sign block B.

This would "reveal" the private key of X+Y, but not X's key or Y's key individually right?

Reply:
It should be still possible to extract the private key for X and Y by using the two sigs received from each of them.

Reply:
But not if the signature is returned summed together in a block, for example in a call to `getBlocksFrom`.

Anyway it's not something that needs to be dealt with in this PR, but I had this thought for a long while now.

Reply:
We don't use the signature with multiple signatures combined in this extraction process. Note that issue https://github.com/bosagora/agora/issues/408 covers the actual implementation to extract the keys for real.